app/src/main/res/values/strings.xml

```diff
@@ -147,126 +147,126 @@
-    <string name="app_components_switch_switch_label">Switch</string>
+    <string name="app_components_switch_switch_label" translatable="false">Switch</string>
     <string name="app_components_switch_switch_a11y">Switch content description</string>
-    <string name="app_components_switch_switchItem_label">Switch item</string>
+    <string name="app_components_switch_switchItem_label" translatable="false">Switch item</string>
 
     <!-- Components: tag -->
-    <string name="app_components_tag_label">Tag</string>
+    <string name="app_components_tag_label" translatable="false">Tag</string>
     <string name="app_components_tag_description_text">A tag is a small element that shows short info like a label, keyword, or category. It helps users quickly find, group, or understand content.</string>
-    <string name="app_components_tag_textAndBulletLayout_label">Text + bullet</string>
-    <string name="app_components_tag_tag_label">Tag</string>
-    <string name="app_components_tag_inputTag_label">Input tag</string>
-    <string name="app_components_tag_roundedCorners_label">Rounded corners</string>
+    <string name="app_components_tag_textAndBulletLayout_label" translatable="false">Text + bullet</string>
+    <string name="app_components_tag_tag_label" translatable="false">Tag</string>
+    <string name="app_components_tag_inputTag_label" translatable="false">Input tag</string>
+    <string name="app_components_tag_roundedCorners_label" translatable="false">Rounded corners</string>
 
     <!-- Components: text input -->
-    <string name="app_components_textInput_label">Text input</string>
+    <string name="app_components_textInput_label" translatable="false">Text input</string>
     <string name="app_components_textInput_description_text">A Text Input is a user interface component that allows users to enter, edit, or select single-line textual data. It is one of the most fundamental form elements used to capture user input such as names, emails, passwords, or search queries.</string>
-    <string name="app_components_textInput_leadingIcon_label">Leading icon</string>
-    <string name="app_components_textInput_trailingIcon_label">Trailing icon</string>
-    <string name="app_components_textInput_trailingIcon_a11y">Trailing icon content description</string>
-    <string name="app_components_textInput_prefix_label">Prefix</string>
-    <string name="app_components_textInput_suffix_label">Suffix</string>
-    <string name="app_components_textInput_placeholder_label">Placeholder</string>
-    <string name="app_components_textInput_helperLink_label">Helper link</string>
+    <string name="app_components_textInput_leadingIcon_label" translatable="false">Leading icon</string>
+    <string name="app_components_textInput_trailingAction_label" translatable="false">Trailing action</string>
+    <string name="app_components_textInput_trailingAction_a11y">Trailing action content description</string>
```

Comment:
We only change "icon" into "action" in the wordings but we keep "icon" in the APIs, is that right?

Reply:
This was for strings homogenization with iOS and Flutter